syntax = "proto3";

package key_value;

service DoStuff {
    rpc say_hello (HelloRequest) returns (HelloReply) {}
    rpc say_hello_again (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
    int32 pid = 1;
}
message HelloReply {
    string retval = 1;
}